src/orb.yml

```diff
@@ -76,6 +76,10 @@ commands:
         description: Npm cache key
         type: string
         default: 'cache-{{ arch }}-{{ .Branch }}-{{ checksum "package.json" }}'
+      working_directory:
+        descrption: Directory containing package.json
```

Comment:
Typo in description

Reply:
Yup, copy/pasted that one too quickly, thanks!